app/services/fetch_projects.rb

```diff
@@ -60,8 +61,8 @@ def call
 
       project = create_or_update_project(assignable_project)
 
-        team_member.projects << project unless team_member.projects.include?(project)
       end
+        team_member.assignments.create(project: project) unless team_member.projects.include?(project)
```

Comment:
Ah I see how this has come together now in regards to the moving of `team_member.save!`. Should this change belong in https://github.com/dxw/team-dashboard/pull/51/commits/f4af6ac7e7a805f8a1c8d9080cbc34d0dd2b9ae7?

Reply:
I reordered the commits instead.